api/Controllers/MiniUrl.cs

```diff
@@ -59,11 +52,11 @@ public ActionResult<MinifyResponse> Post([FromBody] MinifyRequest body)
     }
     
     [HttpGet("{shortUrl}", Name = "Resolve")]
-    public ActionResult Get([FromRoute] ResolveRequest req)
+    public async Task<RedirectResult> Get([FromRoute] ResolveRequest req)
```

Comment:
Use ActionResult, so you can return 404 as well.

Reply:
done